gatsby-theme-landing-page/src/components/heading.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import * as styles from "./heading.module.css";
+
+export default function Heading({
+  children,
+  as,
+  secondary,
+  className,
+  ...rest
+}) {
+  const baseStyle = !secondary ? styles.heading : styles.secondaryHeading;
+
+  /*
+   Uses `as` prop for heading element. If not provided, defaults to 
+   h2 for standard and h3 for secondary
+  */
+  const Component = as || secondary ? "h3" : "h2";
+  return children ? (
```

Comment:
As a general pattern, you could return early here by checking `if (!children) return false` on line 11, before the rest of the logic in this component